signature/jws/types.go

```diff
@@ -52,3 +91,22 @@ type JwsEnvelope struct {
 	// Base64URL-encoded Signature.
 	Signature string `json:"signature"`
 }
+
+var signatureAlgJWSAlgMap = map[signature.Algorithm]string{
+	signature.AlgorithmPS256: "PS256",
+	signature.AlgorithmPS384: "PS384",
+	signature.AlgorithmPS512: "PS512",
+	signature.AlgorithmES256: "ES256",
+	signature.AlgorithmES384: "ES384",
+	signature.AlgorithmES512: "ES512",
+}
+
+var jwsAlgSignatureAlgMap = reverseMap(signatureAlgJWSAlgMap)
```

Comment:
Do we need a `reverseMap`? Where do you use the `jwsAlgSignatureAlgMap`

Reply:
Yes. When we call SignerInfo(), we need to parse protected header to get the string format algorithm and convert it to be signature.Algorithm and save in SignInfo struct.